packages/venia-ui/lib/components/FilterModal/filterBlock.js

```diff
@@ -20,26 +22,62 @@ const FilterBlock = props => {
         ? classes.list_expanded
         : classes.list_collapsed;
 
+    const itemAriaLabel = formatMessage(
+        {
+            id: 'filterModal.item.ariaLabel',
+            defaultMessage: 'Filter products by "{itemName}".'
+        },
+        {
+            itemName: name
+        }
+    );
+
+    const toggleItemOptionsAriaLabel = isExpanded
+        ? formatMessage(
+              {
+                  id: 'filterModal.item.hideOptions',
+                  defaultMessage: 'Hide "{itemName}" filter item options.'
+              },
+              {
+                  itemName: name
+              }
+          )
+        : formatMessage(
+              {
+                  id: 'filterModal.item.showOptions',
+                  defaultMessage: 'Show "{itemName}" filter item options.'
+              },
+              {
+                  itemName: name
+              }
+          );
+
+    const filterItemOptions = isExpanded && (
```

Comment:
It seems the `listClass` handles the visibility. Is there a reason we should add this extra check? I'd stick with one or the other (probably just class) unless there are a11y implications to having a mounted, but invisible component.

Reply:
Thanks for the question, @sirugh.
The `isExpanded` check is necessary for the a11y. Here is the use case:

![Filter item open/close](https://user-images.githubusercontent.com/13456702/115908036-4d988400-a472-11eb-87d8-5c7bafb576ac.gif)

Please let me know whenever you think that's not necessary.
Thanks.

Reply:
What makes it necessary for a11y? Can't quite tell in the gif.

Reply:
I think I've got your point now. Thanks.

Reply:
I was looking at the wrong part of the code. My bad.

Reply:
So I've re-tested the implementation (the issue goal) after the changes.
This `isExpanded` has a huge impact. I've totally forgotten about it.
Here is the situation: when filter item options are rendered, using `TAB` key doesn't move the focus to the next filter item.
![2021-04-26 20 19 56](https://user-images.githubusercontent.com/13456702/116124615-2db0cc80-a6cd-11eb-901c-fbf11f759a15.gif)

@sirugh Should we keep unmounting the filter item options? What do you suggest here?
Thanks.

Reply:
I don't think I quite follow what is happening or what the problem is. Could you jump on a call or slack thread with me sometime?